plugin-mongodb/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "plugin-mongodb"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+alumet = { path = "../alumet" }
+anyhow = "1.0.93"
+log = "0.4.22"
+serde = "1.0.215"
+tokio = "1.41.1"
+
+[dependencies.mongodb]
+version = "3.1.0"
+features = ["sync"]
```

```suggestion
mongodb = { version = "3.1.0", features = ["sync"] }
```
it's easier to read this way (and more coherent with the whole ecosystem)

---

I didn't even know this was possible.
I'm still a Rust newbie 😃